src/components/AppContent/AppDetailsToggle.vue

```diff
@@ -21,17 +21,31 @@
   -->
 
 <template>
-	<a v-tooltip="title" class="app-details-toggle icon-confirm" href="#" />
+	<Button v-tooltip="title" class="app-details-toggle">
+		<template #icon>
+			<Arrowright :size="20" />
```

Comment:
```suggestion
			<ArrowRight :size="20" />
```

and missing the import